src/components/AssetAccountDetails.tsx

```diff
@@ -39,7 +39,7 @@ export const AssetAccountDetails = ({ assetId: caip19, accountId, route }: Asset
           <AssetTransactionHistory limit={3} assetId={caip19} accountId={accountId} />
         </Stack>
         <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
-          <TradeCard />
+          <TradeCard defaultBuyAssetId={caip19} />
```

Comment:
I know @cjthompson proposed to use this in the previous PR (https://github.com/shapeshift/web/pull/1008#issuecomment-1057679103) but I'm not sure about this one.
I still think this should live component-level, so we don't have to drill that default asset ID down while we actually have a memoized selector doing the exact same thing.

Reply:
To clarify my previous comment: react's reconciliation algorithm makes it so drilling the prop down from the selector a few components up, and re-consuming it on the component itself has the same performance and re-renders, but I believe the latter is cleaner. YMMV, happy with both options.